### PR TITLE
[B5] Migrate schedule demo modal to bootstrap 5

### PR DESCRIPTION
**Pull request by biyeun**
## Technical Summary
I noticed that the schedule demo modal was misbehaving on prelogin pages using a bootstrap 5 template. This fixes the Schedule a Demo modal so that it actually works now.

### Safety story
Currently doesn't work on prelogin bootstrap 5 templates, so this fixes it. Additionally, there are not too many live pages that fall under this category (right now just the licensing pages)

### Automated test coverage
Yes

### QA Plan
Not needed

### Migrations

- [x] The migrations in this code can be safely applied first independently of the code

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

**Review by mjriley — APPROVED** [review]
Looks good, and I appreciate the small commits! It made digesting this much HTML much more approachable. I expect you've tested that the Bootstrap 5 modal looks and behaves like the previous version, however, because that much HTML was still difficult for me to reason about.

**Review by mjriley — CHANGES_REQUESTED** [review]
Amending my previous approval because I didn't see the failing test. Changes are approved once the tests pass

File: corehq/apps/analytics/static/analytix/js/hubspot.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ hqDefine('analytix/js/hubspot', [
             phoneNumberSelector: $form.find('input[name="phone"]'),
             submitCallbackFn: function () {
                 $('#get-demo-cta-success').fadeIn();
-                $('#get-demo-cta-form-content').addClass('hidden');
+                $('#get-demo-cta-form-content').addClass('hidden').addClass('d-none');
```
[the remark this answers is not in the document]
Comment (biyeun) [reply corehq/apps/analytics/static/analytix/js/hubspot.js]: yes, good idea. added

**Comment by biyeun** [comment]
thanks @mjriley, had temporarily left this PR alone while trying to resolve the requirejs stuff from another PR. this should be good to go (i think the tests will be passing soon) :)

**Review by mjriley — APPROVED** [review]

**Pull request merged**